packages/modules/currency/src/services/currency-module-service.ts

```diff
@@ -15,20 +15,15 @@ import { ModulesSdkUtils } from "@medusajs/utils"
 import { Currency } from "@models"
 import { entityNameToLinkableKeysMap, joinerConfig } from "../joiner-config"
 
-const generateMethodForModels = {}
-
 type InjectedDependencies = {
   baseRepository: DAL.RepositoryService
   currencyService: ModulesSdkTypes.IMedusaInternalService<any>
 }
 
 export default class CurrencyModuleService<TEntity extends Currency = Currency>
-  extends ModulesSdkUtils.MedusaService<
-    CurrencyTypes.CurrencyDTO,
-    {
-      Currency: { dto: CurrencyTypes.CurrencyDTO }
-    }
-  >(Currency, generateMethodForModels, entityNameToLinkableKeysMap)
+  extends ModulesSdkUtils.MedusaService<{
```

Comment:
NIT: Maybe we import `MedusaService` directly, a bit easier to read IMO

Reply:
+1, think this should be the convention even though it's an extra import

Reply:
done, for the tests unfortunately pipeline unfortunately we will have to wait for the main branch to be finished